package-lock.json

```diff
@@ -671,13 +671,13 @@
     "qs": {
       "version": "6.5.1",
       "resolved": "https://registry.npmjs.org/qs/-/qs-6.5.1.tgz",
-      "integrity": "sha1-NJzfbu+J7EXBLX1es/wMhwNDptg=",
+      "integrity": "sha512-eRzhrN1WSINYCDCbrz796z37LOe3m5tmW7RQf6oBntukAG1nmovJvhnwHHRMAfeoItc1m2Hk02WER2aQ/iqs+A==",
```

Comment:
why are these changes required?

Reply:
i don't know why. The build does that 🤷‍♀️

Reply:
and there are more changes :) I can roll this back, but every build will recreate this changes.